packages/web-react/src/components/UNSTABLE_EmptyState/demo/EmptyStateDefault.tsx

```diff
@@ -21,7 +21,9 @@ const EmptyStateDefault = () => (
           />
         </svg>
         <div className="docs-Placeholder__text">
-          <Heading size="xsmall">Placeholder</Heading>
+          <Text emphasis="bold" marginBottom="space-0">
```

Comment:
Colors also changed, this should be primary and the text below should be secondary.

Please check the styling once more, it looks like the design changed recently…

![image](https://github.com/user-attachments/assets/3916f326-d9e1-4102-b2a6-e55bc77c7a7b)